src/cosmicqc/analyze.py

```diff
@@ -85,8 +85,10 @@ def identify_outliers(
     for feature in feature_thresholds:
         if feature not in df.columns:
             raise ValueError(f"Feature '{feature}' does not exist in the DataFrame.")
-        outlier_df[f"Z_Score_{feature}"] = scipy_zscore(df[feature])
-        zscore_columns[feature] = f"Z_Score_{feature}"
+        outlier_df[(colname := f"{thresholds_name}.Z_Score.{feature}")] = scipy_zscore(
```

Comment:
Thanks for this question - `:=` helps define an assignment expression, also known as a "walrus operator" 🦭 (because it sorta looks like a sideways walrus with tusks). It allows you to define an inline named reference for use later on. Generally it's useful where you might have a variable which might need to be referenced more than once but doesn't make as much sense to create an additional assignment line all to itself (the reference location for the walrus operator will place it right where it's used).